### PR TITLE
[WIP] Create Register and Forgot Password UI Page

### PR DESCRIPTION
**Pull request by duyhung0311**

### CODE REVIEW

File: frontend/src/feature/forgotpassword/style.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    border-radius: 10px;
+    /* box-shadow: 0px 0px 10px 2px gray; */
+    background-color: white    ;
+}
+.sign-up{
+    width: 80%;
+    height: auto;
+    font-size: 30px;
+    color: black;
+    /* background-color:#007c7e; */
```
Comment (hiepvu0201) [inline frontend/src/feature/forgotpassword/style.css:25]: can you remove this comment?

File: frontend/src/feature/register/style.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    ;
+}
+.form-box{
+    width: 25%;
+    height: auto;
+    display:flex;
+    align-items: center;
+    justify-content: start;
+    border-radius: 10px;
+    /* box-shadow: 0px 0px 10px 2px gray; */
```
Comment (hiepvu0201) [inline frontend/src/feature/register/style.css:17]: can you remove this comment too?

File: frontend/src/feature/forgotpassword/style.css (hunk 1 above)
Comment (duyhung0311) [reply frontend/src/feature/forgotpassword/style.css:25]: yes

File: frontend/src/feature/register/style.css (hunk 2 above)
Comment (duyhung0311) [reply frontend/src/feature/register/style.css:17]: yes

**Pull request merged**